flows/aws/interactive-workflow/deploy-interactive-workflow.py

```diff
@@ -0,0 +1,10 @@
+from prefect import flow
+
+flow.from_source(
+source="https://github.com/Sahiler/interactive-workflow-demo.git",
```

Comment:
We'll want to probably set this up in the dev branch for this repo first to do some fine tuning and then create/test the deployment here https://app.prefect.cloud/account/0ff44498-d380-4d7b-bd68-9b52da03823f/workspace/f579e720-7969-4ab8-93b7-2dfa784903e6/dashboard to start with before we merge into main